homework1_aboutme.html

```diff
@@ -0,0 +1,39 @@
+<!--homework1: about me page-->
+
+<html>
+	<head>
+		<title>About me</title>
+		<link rel="stylesheet" type="text/css" href="homework1_aboutme_styles.css">
+	</head>
+		
+		<body>
+
+
+			<div><hr color="#000000" size="4" width="1175px" align= "center"></div>
```

Comment:
Nice use of the ```<hr>``` tag here.  For next time, instead of using the attributes to style the tag (i.e. give it color, height, width), think about how you would do this in just CSS.